packages/abi-gen/src/utils.ts

```diff
@@ -92,4 +93,24 @@ export const utils = {
             inputs: [],
         };
     },
+    makeOutputFileName(name: string): string {
+        let fileName = toSnakeCase(name);
+        if (fileName === 'z_r_x_token') {
+            fileName = 'zrx_token';
+        }
+        return fileName;
+    },
+    writeOutputFile(filePath: string, renderedTsCode: string): void {
+        fs.writeFileSync(filePath, renderedTsCode);
+    },
+    isOutputFileUpToDate(abiFile: string, outputFile: string): boolean {
+        const abiTime = fs.statSync(abiFile).mtimeMs;
+        try {
+            const outputFileTime = fs.statSync(outputFile).mtimeMs;
+            return outputFileTime > abiTime;
```

Comment:
Should be `greater or equal` no? It's not going to impact the code, since these will never be equal, but logically it is more correct to say `>=`.

Reply:
I modeled this after my knowledge of how GNU make works.  I was just going off of instinct, but I looked it up just now, and make works the way I wrote it here.  From the GNU make manual: "The make utility examines time relationships and shall update those derived files (called targets) that have modified times *earlier than* the modified times of the files (called prerequisites) from which they are derived."  (emphasis mine)

Does that change your mind?  (I'm not advocating for one way or the other, just explaining my rationale; happy to defer to your judgement.)

Reply:
Thanks for the explanation! Now that I think about it again, I actually think your solution is more correct. If they both have the same modified timestamp, there would have been no time for the output to be adjusted given the new ABI file (hinting at them being modified independently). Thus, the output would be out of date and would need to be re-generated given the new ABI. Let's keep it how it is.